field.go

```diff
@@ -368,7 +368,22 @@ func Stack(key string) Field {
 	// from expanding the zapcore.Field union struct to include a byte slice. Since
 	// taking a stacktrace is already so expensive (~10us), the extra allocation
 	// is okay.
-	return String(key, takeStacktrace())
+	return String(key, takeStacktrace(0))
+}
+
+// StackSkip constructs a field similarly to Stack, but also skips the given
+// number of frames from the top of the stacktrace.
+func StackSkip(key string, skip int) Field {
+	// Skip the call to StackSkip
+	if skip != 0 {
+		skip++
```

Comment:
minor stylistic thing: Comments on their own lines should be full sentences ending with periods, or they can be inline if they're not full sentences.

```suggestion
	// Skip the call to StackSkip.
	if skip != 0 {
		skip++
```

Or,

```suggestion
	if skip != 0 {
		skip++ // skip StackSkip
```